README.md

```diff
@@ -1,191 +1,122 @@
+> uses SQLite for this purpose.
 
-## Local Development Setup (either Mac or Linux)
+To start Lumigator locally, follow these steps:
 
-1. `git clone git@github.com:mozilla-ai/lumigator.git`
-2. `make local-up`. For more on `docker-compose`, see the [local install documentation.](/.devcontainer/README.md).
-3. To shut down app, `make local-down`
+1. Clone the Lumigator repository:
 
-When running Lumigator locally, `.env.example` is provided as a central source for all environment variables. The `make local-up` command will auto-create a `.env` file based on `.env.example` if it doesn’t exist.
+    ```bash
+    git clone git@github.com:mozilla-ai/lumigator.git
+    ```
 
-### Dev Environment Details
+1. Navigate to the repository root directory:
 
-We use `uv` to manage dependencies. Each project under `.../lumigator/lumigator/python/mzai/` is an independent `uv` Python project to isolate dependencies.  Sub-projects are linked together using editable Python package installs.
+    ```bash
+    cd lumigator
+    ```
 
-For each project, here are some handy `uv` commands to work with the repo
+1. Start Lumigator using Docker Compose:
 
-Change directory to the project you want to work on  (ie. `lumigator/lumigator/python/mzai/backend`)
+    ```bash
+    make start-lumigator
+    ```
 
-**Grab dependencies**
+To verify that Lumigator is running, open a web browser and navigate to
+[`http://localhost:8000`](http://localhost:8000). You should get the following response:
 
-```
-uv sync
+```json
+{"Hello": "Lumigator!🐊"}
 ```
 
-**Run Tests**
+Despite the fact this is a local setup, it lends itself to more distributed scenarios. For instance,
+one could provide different `AWS_*` environment variables to the backend container to connect to any
+provider’s S3-compatible service, instead of localstack. Similarly, one could provide a different
+`RAY_HEAD_NODE_HOST` to move compute to a remote ray cluster, and so on. See
+[here](https://github.com/mozilla-ai/lumigator/blob/7be2518ec8c6bc59ab8463fc7c39aad078bbb386/docker-compose.external.yaml)
+for an example of how to do this, and see the
+[operational guides](https://mozilla-ai.github.io/lumigator/operation-guides/kubernetes.html) in the
+documentation for more deployment options.
 
-```
-uv run pytest
-```
-
-**Add Dependencies to a given project**
-
-```
-uv add package
-```
+Now that Lumigator is running, you can start using it. The platform provides a REST API that allows
+you to interact with the system. Run the [example notebook](/notebooks/walkthrough.ipynb) for a
+quick walkthrough.
 
-Make sure to commit the updated uv.lock file
-
-**Run Test Suite Across all projects**
-```
-make test
-```
-
-**Run the app locally via docker compose:**
+To uninstall Lumigator, simply run the following command:
 
 ```bash
-make local-up
-make local-logs # gets the logs from docker compose
-make local-down # shuts it down
+make stop-lumigator
```

Comment:
I'd say "to stop Lumigator", since we're not uninstalling - or instruct how to truly purge anything Lumigator after stopping (rm the folder). WDYT?

Reply:
@ividal good point. I'll rephrase it to "To stop the containers you started using Docker Compose, simply run the following command:" to make it more explicit. WDYT?